config/api.yml

```diff
@@ -35,6 +35,13 @@
     - :post
     - :patch
     - :delete
+    :gpppdo: &gpppdo
```

Comment:
I'm not sure about this direction. I think we should support `OPTIONS` for every URI. With the upcoming OO controllers, it should be straightforward to route an `OPTIONS` call to the right controller, which can either override `#options` or fall back on the default behavior.